pkg/capr/planner/planner.go

```diff
@@ -1021,44 +1031,58 @@ func (p *Planner) generatePlanWithConfigFiles(controlPlane *rkev1.RKEControlPlan
 		nodePlan plan.NodePlan
 		err      error
 	)
-	if !controlPlane.Spec.UnmanagedConfig {
-		nodePlan, reg, err = p.commonNodePlan(controlPlane, plan.NodePlan{})
-		if err != nil {
-			return nodePlan, map[string]interface{}{}, "", err
-		}
-		var (
-			joinedServer string
-			config       map[string]interface{}
-		)
-		nodePlan, config, joinedServer, err = p.addConfigFile(nodePlan, controlPlane, entry, tokensSecret, joinServer, reg, renderS3)
-		if err != nil {
-			return nodePlan, config, joinedServer, err
-		}
 
-		nodePlan, err = p.addManifests(nodePlan, controlPlane, entry)
-		if err != nil {
-			return nodePlan, config, joinedServer, err
-		}
+	if controlPlane.Spec.UnmanagedConfig {
```

Comment:
was the inversion of the logic here to try and remove indentation? 

from a safety perspective, I think we should just leave it the way it was. we need to see if there are edge cases that don't get caught by this